softdevice_controller/include/ble_controller_soc.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef BLE_CONTROLLER_SOC_H__
+#define BLE_CONTROLLER_SOC_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "sdc_soc.h"
+
+#define BLE_CONTROLLER_FLASH_CMD_STATUS         SDC_FLASH_CMD_STATUS
+#define BLE_CONTROLLER_FLASH_CMD_STATUS_SUCCESS SDC_FLASH_CMD_STATUS_SUCCESS
+#define BLE_CONTROLLER_FLASH_CMD_STATUS_TIMEOUT SDC_FLASH_CMD_STATUS_TIMEOUT
+
+#define ble_controller_flash_callback_t sdc_flash_callback_t
+
+#define ble_controller_flash_write              sdc_flash_write
+#define ble_controller_flash_page_erase         sdc_flash_page_erase
+#define ble_controller_rand_vector_get          sdc_rand_vector_get
+#define ble_controller_rand_vector_get_blocking sdc_rand_vector_get_blocking
+#define ble_controller_ecb_block_encrypt        sdc_ecb_block_encrypt
```

Comment:
I think we can take the opportunity now and align these names with other function names in NCS, and use `_poll` or `_async`, instead of `_blocking`.

```suggestion
#define ble_controller_flash_write              sdc_flash_write_async
#define ble_controller_flash_page_erase         sdc_flash_page_erase_async
#define ble_controller_rand_vector_get          sdc_rand_vector_get_poll
#define ble_controller_rand_vector_get_blocking sdc_rand_vector_get
#define ble_controller_ecb_block_encrypt        sdc_ecb_block_encrypt
```

Reply:
We should also use the prefix `sdc_soc_` instead of `sdc_` only

Reply:
Done